reflection.md

```diff
@@ -2,17 +2,71 @@
 
 ## Which operators were easiest to implement and why?
 
+The `||` and `~` operators were easiest to implement (after figuring out the
+implicit conversion code, see below) since they closely resembled the sample we
+did in class with `add`/`+`. Their implementation was also a simple one-to-one call
+on the predefined case classes which contributed to their level of ease.
+
 ## Which operators were most difficult to implement and why?
 
+Although not directly an operator, the implicit conversion code (`string2regularExpression`
```

Comment:
I also initially had this problem and figured it out by looking at the final code for Program.scala was provided in teh write-up.